code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -146,7 +146,7 @@
 
 	if(istype(W,/obj/item/weapon/pen)) //Renaming food
 		var/n_name = copytext(sanitize(input(user, "What would you like to name this dish?", "Food Renaming", null)  as text), 1, MAX_NAME_LEN)

```

Comment:
probably want as null|anything after this as well just so people can cancel the action.